swan-lake/integration-tools/openapi-tool.md

```diff
@@ -46,17 +46,43 @@ $ bal openapi [-i | --input] <ballerina-service-file-path> [--json]
             [-o | --output] <output-location>
 ```
 
+### Sanitize OpenAPI
+
+The Ballerina to OpenAPI command supports several usages in the Ballerina OpenAPI tool as follows.
```

Comment:
@SachinAkash01 This is a CLI that accepts an OpenAPI specification as input and generates a sanitized OpenAPI specification as output, shall we change this description according to it?